src/main/java/javafx/util/Pair.java

```diff
@@ -0,0 +1,65 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by FernFlower decompiler)
+//
+
+package javafx.util;
+
+import java.io.Serializable;
+
+public class Pair<K, V> implements Serializable {
```

Comment:
I saw you have `Lombok` for this project. Does it make sense just annotate `@Value`?